Godeps/Godeps.json

```diff
@@ -1,7 +1,6 @@
 {
 	"ImportPath": "github.com/opencontainers/runtime-tools",
 	"GoVersion": "go1.7",
-	"GodepVersion": "v79",
```

Comment:
Where did this go?  Do newer `godep` no longer set it?

Reply:
Actually I updated the package with `godep` and it was deleted.